script/ts/configs/LiquidityHandler/set-hlp-staking.ts

```diff
@@ -0,0 +1,29 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { ethers } from "hardhat";
```

Comment:
`import { ethers } form "ethers"` instead

Reply:
There is no `getSigners()` in that ethers.